src/utils/hederaUtils.js

```diff
@@ -156,24 +145,35 @@ module.exports = class HederaUtils {
           new HederaSDK.Hbar(-10000)
         );
       const tx = await transferTransaction.execute(client);
-      await tx.getReceipt(client);
-
-      const accountInfo = await new HederaSDK.AccountInfoQuery({
-        accountId: HederaSDK.AccountId.fromEvmAddress(0, 0, wallet.address),
-      }).execute(client);
+      if (startup) {
+        logger.log(
+          `| 0.0.${firstAccountNum} - ${wallet.address} - ${
+            wallet._signingKey().privateKey
+          } - ${new HederaSDK.Hbar(10000)} |`
```

Comment:
make HBAR value const. Seems to be used at least 4 times.
In 2 places we use tinybar.

Reply:
why are we using both` Hbar(10000)` and in some placces: `Hbar.fromTinybars(10000000000000)`
should it be the same right? could we get away with always using Hbar(initialBalanceConst)

Reply:
Not sure honestly, but I will change it to be the same everywhere.